karamel-core/src/main/java/se/kth/karamel/cookbook/metadata/KaramelizedCookbook.java

```diff
@@ -24,25 +30,121 @@
   private final DefaultRb defaultRb;
   private final MetadataRb metadataRb;
   private final KaramelFile karamelFile;
-  private final Berksfile berksfile;
+  private final Berksfile berksFile;
+  private List<ExperimentRecipe> experimentRecipes = new ArrayList<>();
+  private InstallRecipe installRecipe;
   private String json;
 
-  public KaramelizedCookbook(String homeUrl) throws CookbookUrlException, MetadataParseException {
+  /**
+   *
+   * @param homeUrl url or canonical path to the cookbook
+   * @param local true if it is a canonical path (to a cloned cookbook) and not a URL.
+   * @throws CookbookUrlException
+   * @throws MetadataParseException
+   */
+  public KaramelizedCookbook(String homeUrl, boolean local) throws CookbookUrlException, MetadataParseException {
+    if (local) {
```

Comment:
It's a hack, TBH. You had already written KaramelizedCookbook, which uses direct URLs, and it was quick and easier for me to modify it. Then i realized that everytime i load a cookbook, i clear my local copy, clone it from GitHub, and then KaramelizedCookbook reads the files (again) from GitHub - using URLs, not the GitHub APIs. The local option, basically means - don't read using URLs, read from the local clone. This could be rewritten, but I really don't have time right now. I think there's tests for it, though...